src/clientSideScene/CameraControls.ts

```diff
@@ -773,6 +773,80 @@ export class CameraControls {
     })
   }
 
+  async updateCameraToAxis(
+    axis: 'x' | 'y' | 'z' | '-x' | '-y' | '-z' | 'reset'
+  ): Promise<void> {
+    if (axis === 'reset') {
+      await this.resetCameraPosition()
+      return
+    }
+
+    const distance = this.camera.position.distanceTo(this.target)
+
+    let vantage = { x: 0, y: 0, z: 0 }
+    let up = { x: 0, y: 0, z: 0 }
+
+    if (axis === 'x') {
+      vantage = { x: distance, y: 0, z: 0 }
```

Comment:
Could we change this so to something like 
```typescript
    const vantage = this.target.clone()
    let up = { x: 0, y: 0, z: 0 }

    if (axis === 'x') {
      vantage.x += distance
      up = { x: 0, y: 0, z: 1 }
// ...
```
And then for the default camera look at keep the users's target
```typescript
await this.engineCommandManager.sendSceneCommand({
      type: 'modeling_cmd_req',
      cmd_id: uuidv4(),
      cmd: {
        type: 'default_camera_look_at',
        center: this.target,
        vantage: vantage,
        up: up,
      },
    })
```

Sorry if if this is different from what I originally asked for, but think this will be better as it respects the user's current target, and just rotates the camera to a different perspective.

Reply:
now I can see that too ! so much better to code with viewport on :DDD